python/pyarrow/dataset.py

```diff
@@ -848,9 +848,9 @@ def _ensure_write_partitioning(part, schema, flavor):
 
 
 def write_dataset(data, base_dir, *, basename_template=None, format=None,
-                  partitioning=None, partitioning_flavor=None, schema=None,
-                  filesystem=None, file_options=None, use_threads=True,
```

Comment:
Is this a breaking change? I.e., it inserts a new parameter in between the existing parameter list.